### DISCUSSION TITLE
Is 2024.32.x turning off USB ports for anyone else?

### DISCUSSION THREAD

**Discussion by mattenz** [marcone/teslausb#879] [General]
I have a 2019 Model 3 and it seems since 2024.32.3 a few weeks ago, whenever my car is parked at home with no one inside it, it will turn off the USB ports soon after all doors are closed, even if the car is awake and charging. This obviously means that TeslaUSB is often not able to be online long enough to finish uploading a busy days worth of content.

I never bothered setting up the API connection so Sentry would be turned on until archiving was completed but it seems like that is going to be mandatory now with this update.

**Comment by lbr88** [comment]
This is a known issue with this version. It's also a problem for the sexy commander because it turns off the entire 12V system as soon as you leave the car. It's unknown if it's a bug and whether or not Tesla is going to release a fix or if it is intended.

**Reply by mattenz** [reply]
Based on this recent update to the manual, it seems like it’s an intended new change for when the car is left unattended, but it seems like a bug that it’s also happening when the car is charging:
`Power is available when the vehicle detects that a user is present. A user is considered present when a person is in the driver seat or interacts with the touchscreen. Power is also available whenever the low voltage battery is being charged or is in use, when you charge the high voltage Battery, or when the vehicle is communicating with the mobile app. Leaving an accessory plugged in does not deplete the low voltage battery. `

https://www.tesla.com/ownersmanual/2017_2023_model3/en_nz/GUID-7F07443D-5107-4A5A-A9F1-E02FF14E4A9A.html
